python/tvm/topi/adreno/conv2d_nhwc.py

```diff
@@ -270,37 +264,27 @@ def schedule_conv2d_NHWC(cfg, s, output):
     ##### space definition end #####
 
     pad_data, kernel = s[conv].op.input_tensors
-    if (
-        isinstance(kernel.op, tvm.te.ComputeOp) and "filter_pack" in kernel.op.tag
-    ):  # len(latest.op.axis) == 4:
-        # manage scheduling of datacopy
-        pad_data, kernel = s[conv].op.input_tensors
+    if autotvm.GLOBAL_SCOPE.in_tuning or input_pack_rt:
         if "pad_temp" in pad_data.op.name:
+            s[pad_data].compute_inline()
```

Comment:
I read this as: If there is input packing we inline the padding into the packing step. Is this correct?

Reply:
there is several conditions which I was not able to form in more convenient way. I have added comment when we add cache_read or schedule other stages on the data flow